New LWC Quick Start (#4838)
forcedotcom/salesforcedx-vscode: docs/_articles/en/localdev/lwclocaldev.md

The Local Development server uses the following Salesforce CLI commands.

> View the commands from VS Code's Command Palette by pressing `command` + `shift` + `p`. By typing in "local development," all three commands are visible.

Reviewer: ```suggestion
View the commands from VS Code's Command Palette by pressing `command` + `shift` + `p`. If you type "local development," all three commands are visible.
```
